commands/init.go

```diff
@@ -45,13 +45,14 @@ func transformInitArgs(args *Args) error {
 		err    error
 	)
 
-	if args.IsParamsEmpty() {
+	words := args.Words()
```

Comment:
[`args.Words()'](https://github.com/github/hub/blob/master/commands/args.go#L23-L32) returns args not starting with "-"